modules/fontrendering/src/textrenderer.cpp

```diff
@@ -40,6 +40,7 @@
 #include <modules/opengl/openglutils.h>
 #include <modules/opengl/shader/shaderutils.h>
 #include <modules/opengl/texture/textureutils.h>
+#include <modules/opengl/texture/texture2d.h>                        // for Texture2D
```

Comment:
remove comment

Reply:
We might add these comment for every include in the future, or not, we should discuss that.